cmd/minikube/cmd/start_test.go

```diff
@@ -483,11 +483,7 @@ func TestValidateWindowsOSVersion(t *testing.T) {
 		errorMsg  string
 	}{
 		{
-			osVersion: "2019",
-			errorMsg:  "",
-		},
-		{
-			osVersion: "2022",
+			osVersion: "2025",
```

Comment:
Invalid error message provides 2019 and 2022 as valid OS versions instead of 2025, see line 491